aws/table_aws_ses_domain_identity.go

```diff
@@ -0,0 +1,186 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ses"
+	"github.com/turbot/go-kit/types"
+	"github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin/transform"
+)
+
+func tableAwsSESDomainIdentity(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_ses_domain_identity",
+		Description: "AWS SES Domain Identity",
+		List: &plugin.ListConfig{
+			Hydrate: listSESDomainIdentities,
+		},
+		GetMatrixItem: BuildRegionList,
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "identity",
+				Description: "The domain identity.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromValue(),
+			},
+			{
+				Name:        "arn",
+				Description: "The ARN of the AWS SES identity.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getDomainIdentityARN,
+				Transform:   transform.FromValue(),
+			},
+			{
+				Name:        "verification_status",
+				Description: "The verification status of the identity.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getDomainIdentityVerificationAttributes,
+				Transform:   transform.FromField("VerificationStatus"),
+			},
+			{
+				Name:        "verification_token",
+				Description: "The verification token for a domain identity.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getDomainIdentityVerificationAttributes,
+				Transform:   transform.FromField("VerificationToken"),
+			},
+			{
+				Name:        "notification_attributes",
+				Description: "Represents the notification attributes of an identity.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getDomainIdentityNotificationAttributes,
+				Transform:   transform.FromValue(),
+			},
+
+			// Standard columns for all tables
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromValue(),
+			},
+			{
+				Name:        "akas",
+				Description: resourceInterfaceDescription("akas"),
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getDomainIdentityARN,
+				Transform:   transform.FromValue().Transform(transform.EnsureStringArray),
+			},
+		}),
+	}
+}
+
+//// LIST FUNCTION
+
+func listSESDomainIdentities(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	logger := plugin.Logger(ctx)
+	logger.Trace("listSESDomainIdentities")
+	region := d.KeyColumnQualString(matrixKeyRegion)
+
+	// Create Session
+	svc, err := SESService(ctx, d, region)
+	if err != nil {
+		return nil, err
+	}
+
+	input := &ses.ListIdentitiesInput{
+		MaxItems:     aws.Int64(1000),
+		IdentityType: aws.String(ses.IdentityTypeDomain),
```

Comment:
I didn't know they had these constants, but thanks for using it/letting us know, we'll update the other table to use the email equivalent as well.

Reply:
I also updated the ses_email_identity table to use the constant